docs/source/resources/usage_tips/glossary.rst

```diff
@@ -29,8 +29,8 @@ Glossary
     instance
         Equivalent to a row in a relational database. Each entity has many instances, and each instance has a value for each variable and feature defined on the entity.
 
-    target entity
-        The entity on which we will be making a features for.
+    target dataframe
+        The dataframe on which we will be making a features for.
```

Comment:
Not related to your update, but maybe we should clean up the wording while we are making changes here:
`The dataframe for which we will be making features`

Reply:
updated